test/entrypoint/srv_empty.lua

```diff
@@ -9,6 +9,9 @@ local ok, err = remote_control.bind(
 )
 assert(ok, err)
 
+ok, err = require('cartridge').cfg({}, {log = 'stderr'})
+assert(ok, err)
```

Comment:
Is it possible to configure it with `log.cfg/box.cfg`. I'm not sure that direct cartridge.cfg call will work as expected.

`cartridge.cfg` doesn't work in the same way as `box.cfg`. While two sequential `box.cfg{}` can reload different parts of configuration (e.g. `replication` and `log`) `cartridge.cfg` updates whole configuration.

Reply:
log.cfg can only be called once (it reports an error if called more then once trying to set `log` option). And `cartridge.cfg` calls it). There are two solutions to that:
1) Add the `log` option only to existent `cartridge.cfg` calls (currently implemented)
2) Set env variable `TARANTOOL_LOG=""`

I don't know which solution I like more, whats your opinion on that?

Reply:
Well, let's stay as is since it works